authn.js

```diff
@@ -211,18 +211,16 @@ function setup(app) {
 
   // Provide the client-side app with info about the current user
   app.route("/whoami").get((req, res) => {
+    if (req.user && req.user.groups) {
+      // Filter user groups to match available Nextstrain sources
+      req.user.groups = req.user.groups.filter((group) => sources.has(group));
```

Comment:
[Just made some comments](https://github.com/nextstrain/nextstrain.org/commit/1a65e10b0094e74642668cea2abd58ae91420c19#commitcomment-36703817) on the commit addressing this review note. Cross-linking now for completeness. I couldn't initially find this review note because resolved notes are collapsed by default, and I missed it.